prow/cmd/build/controller.go

```diff
@@ -94,37 +127,38 @@ func newController(kc kubernetes.Interface, pjc prowjobset.Interface, pji prowjo
 
 	// Create struct
 	c := &controller{
-		pjc:        pjc,
-		builds:     buildConfigs,
-		pjLister:   pji.Lister(),
-		pjInformer: pji.Informer(),
-		workqueue:  workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), controllerName),
-		recorder:   recorder,
-		totURL:     totURL,
+		pjc:         pjc,
+		builds:      buildConfigs,
+		pjLister:    pji.Lister(),
+		pjInformer:  pji.Informer(),
+		workqueue:   workqueue.NewNamedRateLimitingQueue(limiter(), controllerName),
```

Comment:
feels like this should be passed in from main, like other fields.

Reply:
Done